src/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const spaRoot = window["getOpenmrsSpaBase"];
+export const basePath = "/dispensing";
+export const spaBasePath = `${window.spaBase}${basePath}`;
```

Comment:
Let's call this `dispensingBasePath`. `spaBasePath` is `/openmrs/spa`.

Reply:
Done